plugins/providers/metabase/config.go

```diff
@@ -170,6 +173,10 @@ func (c *Config) validatePermission(resourceType string, value interface{}) (*Pe
 		nameValidation = "oneof=schemas:all native:write"
 	} else if resourceType == ResourceTypeCollection {
 		nameValidation = "oneof=read write"
+	} else if resourceType == ResourceTypeTable {
+		nameValidation = "oneof=all"
+	} else if resourceType == ResourceTypeGroup {
+		nameValidation = "len=0"
```

Comment:
when creating an appeal, we are asked for `email`, `resource_id`, and `role`. This makes we're allowed to enter empty `role` (`""`) which I think could be confusing. What do you think if we add one role like `member` or other else (open for discussion) for metabase group? so when creating an appeal the `role` value won't be empty